### ISSUE TITLE
Errors when running the code.

### ISSUE THREAD

**Issue by dickson-wong** [YingGwan/collisionAwareSOROSimulator#1]
I am Dickson. I am trying to use the simulator for our soft actuator. 

I have Visual Studio and Qt 6.7.2 installed in Windows 11. I am following the instructions on the README page to replicate the simulation. However, there are errors when I run the program. Here's what I did:

1. Install Visual Studio 2022 and Qt 6.7.2.
2. Install the "Qt Visual Studio Tools" in Visual Studio 2022.
3. Add the "Qt Version" and "Path."
4. Open the local folder "collisionAwareSOROSimulator-main"
5. After that, I follow these remaining instructions.
<img width="624" alt="image" src="https://github.com/user-attachments/assets/c86ee438-0441-4993-8ee7-ba16b8fbb012">

I run in release mode. Finally, when I run the program, errors occur. I have attached the error message here.
[Output-Build.txt](https://github.com/user-attachments/files/16988575/Output-Build.txt)

Could you please help me with that?

Thank you.

**Comment by YingGwan** [comment]
Hi Dickson, could you install the QT with version 5.12.3. Remember to install QtWidgets during installation.
The link: https://download.qt.io/archive/qt/5.12/5.12.3/
![image](https://github.com/user-attachments/assets/0172056e-84be-47f6-97da-6256fc86ca7e)

**Comment by dickson-wong** [comment]
Thanks for the help. we can finally run the simulator. By the way, may I ask how to convert the Solidworks CAD file to tet file and then import to the simulator?

**Comment by YingGwan** [comment]
Hi could you summarize your tasks and send the surface meshes (one is outer surface mesh while one is chamber surface mesh) to us?

**Comment by dickson-wong** [comment]
Hi, is it possible to have a video meeting to discuss this in detail?

**Comment by YingGwan** [comment]
No problem, please send invitation to this email address: yingjun.tian@postgrad.manchester.ac.uk
